docs/api/datasets.rst

```diff
@@ -93,6 +93,12 @@ iNaturalist
 
 .. autoclass:: INaturalist
 
+LandCover.ai
```

Comment:
This name will need to be unique, this is the link that the names in the CSV files search for